Astro CLI v1.0.0 (All Changes) (#690)
astronomer/docs: astro/cli/release-notes.md

---
sidebar_label: 'Astro CLI'
title: 'Astro CLI Release Notes'
id: cli-release-notes
description: Release notes for the Astro CLI.
---

## Overview

This document provides a summary of all changes made to the [Astro CLI](install-cli.md). For general product release notes, go to [Astro Release Notes](release-notes.md).

> If you have any questions or a bug to report, don't hesitate to reach out to us via Slack or Intercom. We're here to help.

Reviewer: ```suggestion
If you have any questions or a bug to report, reach out to [Astronomer Support](https://support.astronomer.io) We're here to help.
```

## Astro CLI v1.0.0

Release date: May 27, 2022

### A Shared CLI for All Astronomer Users

:::danger Breaking Change

Astro CLI v1.0.0 includes breaking changes that might effect your existing CI/CD pipelines. Ensure that all of your CI/CD pipelines are updated to use the new CLI executable as described below before upgrading.

:::

The Astro CLI is now a single CLI executable built for all Astronomer products. This new generation of the CLI optimizes for a consistent local experience with Astro Runtime as well as the ability to more easily upgrade to Astro from other products hosted on Astronomer.

To establish a shared framework between products, the Astro CLI now uses a single `astro` executable:

```sh
# Before upgrade
astrocloud dev init

# After upgrade
astro dev init
```

Additionally, some commands have been standardized so that they can be shared between Astro and Astronomer Software users. As part of this change, `astro auth login` and `astro auth logout` have been renamed to `astro login` and `astro logout`:

```sh
# Before upgrade
astrocloud auth login

# After upgrade
astro login
```

> For Astro users, these are the only changes to existing CLI functionality. All other commands will continue to work as expected. We strongly recommend that all users upgrade. For instructions, see [Configure the CLI](cli/configure-cli.md).

Reviewer: ```suggestion
For Astro users, these are the only changes to existing CLI functionality. All other commands will continue to work as expected. We strongly recommend that all users upgrade. For instructions, see [Migrate from `astrocloud` to `astro`](cli/configure-cli.md#migrate-from-astrocloid-to-astro).
```

---
sidebar_label: 'Astro CLI'
title: 'Astro CLI Release Notes'
id: cli-release-notes
description: Release notes for the Astro CLI.
---

## Overview

This document provides a summary of all changes made to the [Astro CLI](install-cli.md). For general product release notes, go to [Astro Release Notes](release-notes.md).

If you have any questions or a bug to report, reach out to [Astronomer Support](https://support.astronomer.io)

## Astro CLI v1.0.0

Release date: May 27, 2022

### A Shared CLI for All Astronomer Users

The Astro CLI is now a single CLI executable built for all Astronomer products. This new generation of the CLI optimizes for a consistent local experience with Astro Runtime as well as the ability to more easily upgrade to Astro from other products hosted on Astronomer.

To establish a shared framework between products, the Astro CLI now uses a single `astro` executable:

```sh
# Before upgrade
astrocloud dev init

# After upgrade
astro dev init
```

> Additionally, some commands have been standardized so that they can be shared between Astro and Astronomer Software users. As part of this change, `astro auth login` and `astro auth logout` have been renamed to `astro login` and `astro logout`:

Reviewer: ```suggestion
Additionally, some commands have been modified for simplicity. As part of Astro CLI v1.0.0, `astro auth login` and `astro auth logout` have been renamed to `astro login` and `astro logout`:
```

```sh
# Before upgrade
astrocloud auth login

# After upgrade
astro login
```

For Astro users, these are the only changes to existing CLI functionality. All other commands will continue to work as expected. We strongly recommend that all users upgrade. For instructions, see [Migrate from `astrocloud` to `astro`](cli/configure-cli.md#migrate-from-astrocloud-to-astro).

:::caution Possible Breaking Change

> Upgrading to Astro CLI v1.0.0 includes breaking changes that might effect your existing CI/CD pipelines. Ensure that all of your CI/CD pipelines are updated to use the new CLI executable before upgrading.

Reviewer: ```suggestion
If you currently have CI/CD pipelines that install the `astrocloud` executable of the Astro CLI, we encourage you to update them to use the latest version of `astro` to ensure reliability. All `astrocloud` commands will continue to work for some time but will be deprecated by Astronomer soon.

For updated CI/CD examples, see [CI/CD](ci-cd.md).
```
